src/components/Person/Person.jsx

```diff
@@ -1 +1,26 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const sex = person.sex === 'm' ? 'wife' : 'husband';
+
```

Comment:
CL# 2

Reply:
done!)
but, in my opinion, if the solution is a short line - it is normal to write everything in one line.